webpack.config.js

```diff
@@ -63,12 +63,9 @@ module.exports = {
         test: /\.(js|jsx)$/,
         include: path.resolve(__dirname, 'src'),
         // TODO: because it uses entry point to determine graph of dependencies, might not be needed to exclude test ans sample files?
-        exclude: /(node_modules|bower_components|build|dist|demo|.storybook|storybook-static)/,
+        exclude: /(node_modules|bower_components|build|dist|demo|storybook-static)/,
```

Comment:
```suggestion
```

Actually, testing this out - we don't need the exclusion at all. The src file doesn't contain any of that anyway.

Reply:
ah, cool, good catch

Reply:
it was probably just babel and the node version